packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -434,6 +434,10 @@ const imageNodeType = ({
         description: stripIndent`
         If set, the height of the generated image. If omitted, it is calculated from the supplied width, matching the aspect ratio of the source image.`,
       },
+      aspectRatio: {
+        type: GraphQLFloat,
+        description: stripIndent`If set, the image will be cropped using the width of the generated image in relation to the provided aspect ratio.`,
```

Comment:
```suggestion
        description: stripIndent`
        If set along with width or height, this will set the value of the other dimension to match the provided aspect ratio, cropping the image if needed. 
        If neither width or height is provided, height will be set based on the intrinsic width of the source image.
        `,
```